src/plugins/intel_cpu/src/nodes/scatter_update.cpp

```diff
@@ -56,47 +56,147 @@ ScatterUpdate::ScatterUpdate(const std::shared_ptr<ov::Node>& op, const GraphCon
+                reduction_type = ScatterUpdate::Reduction::SUM;
+                break;
+            case OpReduction::PROD:
+                reduction_type = ScatterUpdate::Reduction::PROD;
+                break;
+            case OpReduction::MEAN:
+                reduction_type = ScatterUpdate::Reduction::MEAN;
+                break;
+            case OpReduction::MAX:
+                reduction_type = ScatterUpdate::Reduction::MAX;
+                break;
+            case OpReduction::MIN:
+                reduction_type = ScatterUpdate::Reduction::MIN;
+                break;
+            case OpReduction::NONE:
+                reduction_type = ScatterUpdate::Reduction::NONE;
+                break;
+            default:
+                THROW_CPU_NODE_ERR("ScatterElementsUpdate CPU does not support reduction mode: ", ov::as_string(node_element->get_reduction()));
+        }
+        use_init_val = node_element->get_use_init_val();
+    } else if (const auto node_element = ov::as_type_ptr<const ov::op::v15::ScatterNDUpdate>(op)) {
+        using OpReduction = ov::op::v15::ScatterNDUpdate::Reduction;
+        switch (node_element->get_reduction()) {
+            case OpReduction::SUM:
+                reduction_type = ScatterUpdate::Reduction::SUM;
+                break;
+            case OpReduction::PROD:
+                reduction_type = ScatterUpdate::Reduction::PROD;
+                break;
+            case OpReduction::SUB:
+                reduction_type = ScatterUpdate::Reduction::SUB;
+                break;
+            case OpReduction::MAX:
+                reduction_type = ScatterUpdate::Reduction::MAX;
+                break;
+            case OpReduction::MIN:
+                reduction_type = ScatterUpdate::Reduction::MIN;
+                break;
+            case OpReduction::NONE:
+                reduction_type = ScatterUpdate::Reduction::NONE;
+                break;
+            default:
+                THROW_CPU_NODE_ERR("ScatterNDUpdate CPU does not support reduction mode: ", ov::as_string(node_element->get_reduction()));
+        }
     }
 }
 
+void ScatterUpdate::getSupportedDescriptors() {
+    if ((getParentEdges().size() != 3) && (getParentEdges().size() != 4))
+        THROW_CPU_NODE_ERR(errorPrefix, " has incorrect number of input edges");
+    if (getChildEdges().empty())
+        THROW_CPU_NODE_ERR(errorPrefix, " has incorrect number of output edges");
+}
+
 void ScatterUpdate::initSupportedPrimitiveDescriptors() {
     if (!supportedPrimitiveDescriptors.empty())
         return;
+    indicesPrec = getOriginalInputPrecisionAtPort(INDICES_ID);
+    auto indicesType = DnnlExtensionUtils::ElementTypeToDataType(indicesPrec);
+    indicesSize = DnnlExtensionUtils::sizeOfDataType(indicesType);
+    if (indicesSize >= 8) {
+        indicesPrec = ov::element::i64;
+        indicesSize = 8;
+    } else {
+        indicesPrec = ov::element::i32;
+        indicesSize = 4;
+    }
```

Comment:
Why does it need to be overwritten? 
(indicesPrec, indicesSize)

Reply:
This was legacy code, it might appear as new addition due to some refactors